src/App.js

```diff
@@ -1,25 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='Body'>
```

Comment:
Não acho que a gente vá precisar de uma caixona maior pra agrupar tudo.
Pense que a página do Navegador já é onde a gente coloca as coisas.
Adicionar uma div envolta do projeto todo vai dificultar mais do que ajudar

Reply:
Isso não tem a ver com a task exatamente, então se for só a fim de teste, beleza ficar assim por enquanto👍🏼

Reply:
> Não acho que a gente vá precisar de uma caixona maior pra agrupar tudo. Pense que a página do Navegador já é onde a gente coloca as coisas. Adicionar uma div envolta do projeto todo vai dificultar mais do que ajudar

Entendi, mas nesse caso o que você sugere que eu faça? Porque a função so aceita retornar uma div, certo? Tinha sido por isso que fiz uma div maior, para englobar as outras.